kong.conf.default

```diff
@@ -29,6 +29,17 @@
                                  # Each Kong process must have a separate
                                  # working directory.
 
+#port_map =                      # When running Kong behind layer 4 port mapping
+                                 # (or port-forwarding), e.g. with
+                                 # `docker run -p 80:8000`, you can use this
+                                 # parameter to let Kong know about published
+                                 # port when it differs from the one Kong is
```

Comment:
published port is ambiguous because every port at kong or lb is published in some sense.

Reply:
Published port is a term that `docker` uses for `-p`. The port that Kong listens is not published e.g. you can only access it from container. For public it is published by docker at port x. What do you suggest instead?

Reply:
I rewrote the whole thing. Not sure if it is any better.